apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -499,13 +502,21 @@ static void tc_task_task_reparent(void)
 */
 static void tc_task_task_init(void)
 {
+	int ret = 0;
+
 	task_init_flag = false;
 
+	sem_init(&task_sem, 0, 0);
```

Comment:
Is it okay to initialize with 0? Then it will be hanged in line 515. Who will post this sem?